doc/content/microreactors/gcmr/GCMR_Core_Neutronics.md

```diff
@@ -101,11 +93,34 @@ MOOSE's Reactor Module [!citep](shemon2023reactor) was used to create the mesh s
 
 !listing microreactors/gcmr/core/MESH/Griffin_mesh.i
 
+## Cross-Section Generation Using Serpent Code
+
+The first step is to generate homogenized multi-group cross-sections using Serpent-2. The specific version used for this work is **Serpent 2.1.32**. The generated cross-sections are then converted into an XML-format file for compatibility with Griffin.
+
+## Note on LFS files linked to Serpent Model
+
+**Large File Storage (LFS)** is used for the following files:
+```
+PART_U901_PF40_R85
+PART_U902_PF40_R85
+PART_U903_PF40_R85
+PART_U904_PF40_R85
+```
```

Comment:
```suggestion
**Large File Storage (LFS)** is used for the following files:

```
PART_U901_PF40_R85
PART_U902_PF40_R85
PART_U903_PF40_R85
PART_U904_PF40_R85
```

```
see website this does not render right now